clearwater-cassandra/usr/share/clearwater/bin/restore_backup.sh

```diff
@@ -114,5 +117,7 @@ do
   chown cassandra:cassandra $TARGET_DIR/*
 done
 
-service cassandra start
+# Start Cassandra.  We remove any xss=.., as this can be printed out by
+# cassandra-env.sh
+service cassandra start | grep -v "^xss = "
```

Comment:
Can you remove this line entirely (so that monit always controls cassandra starting)?